pkg/relay/ethereum/chain.go

```diff
@@ -156,7 +156,7 @@ func (c *Chain) ChainID() string {
 }
 
 // GetLatestHeight gets the chain for the latest height and returns it
-func (c *Chain) LatestHeight() (ibcexported.Height, error) {
+func (c *Chain) LatestHeight(ctx context.Context) (ibcexported.Height, error) {
 	logger := c.GetChainLogger()
 	bn, err := c.client.BlockNumber(context.TODO())
```

Comment:
Although it is obvious that we should replace `context.TODO()` with `ctx`, I would like to do it along with adding a context argument to private methods in this repository in the fourth phase described on https://github.com/hyperledger-labs/yui-relayer/pull/153.